src/components/DynamicFormContainer/DeleteModal/DeleteModal.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FunctionComponent } from "react";
+import { Button } from "../../../UI/Button";
+import { ModalDialog } from "../../ModalDialog";
+
+interface DeleteModalProps {
+  deleteForm: () => void;
+  show: boolean;
+  closeDeleteModal: () => void;
+}
+
+export const DeleteModal: FunctionComponent<DeleteModalProps> = ({
+  deleteForm,
+  show,
+  closeDeleteModal,
+}) => {
+  return (
+    <>
+      {show && (
```

Comment:
Same thing, can remove show here as well. Managed by whoever calls the dialog

Reply:
erm because we using the useLockBodyScroll, if the component is being rendered but not shown on screen, the body will be locked, so still need the show props in here, but can remove in the modalDialog.